docs/release.md

```diff
@@ -0,0 +1,13 @@
+# Zenko Release Instructions
+
+## Release Process
+
+To release the Zenko and Zenko-base ISOs:
+
+1. Tag and release the Zenko version using the [GitHub release page](https://github.com/scality/Zenko/releases/new).
+1. Retrieve the eve `artifacts_name` of the tagged build. This can be found
+   under the `Build Properties` of the selected eve build.
+1. Tag the selected artifacts using the eve `promote` command:
+   * The `promote` command can be found on the eve builders page.
```

Comment:
Is this a static link? Or is it unique per build? Finding stuff on eve can be hard so using the URL might help in the future.